lib/types/string.js

```diff
@@ -343,10 +359,38 @@ module.exports = Any.extend({
                     options.separator === true ? '[:-]' :
                         options.separator === false ? '[]?' : `\\${options.separator}`;
 
-                const regex = new RegExp(`^([\\[{\\(]?)[0-9A-F]{8}(${separator})[0-9A-F]{4}\\2?[${versionNumbers || '0-9A-F'}][0-9A-F]{3}\\2?[${versionNumbers ? '89AB' : '0-9A-F'}][0-9A-F]{3}\\2?[0-9A-F]{12}([\\]}\\)]?)$`, 'i');
+                let wrapperStart = '';
+                let wrapperEnd = '';
```

Comment:
We can skip the initialization, as all paths will override it anyway
```suggestion
                let wrapperStart;
                let wrapperEnd;
```